From d1163b12a1da183d9e8a8df4180b03afe4ef775d Mon Sep 17 00:00:00 2001
From: Jelle van der Waa <jelle@vdwaa.nl>
Date: Wed, 1 Jan 2020 18:30:49 +0100
Subject: [PATCH] plugins: password: make Python 3 compatible

Remove the ", e" as the exception is never printed and this makes it
Python 3 compatible as well
---
 plugins/password/helpers/chpass-wrapper.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/password/helpers/chpass-wrapper.py b/plugins/password/helpers/chpass-wrapper.py
index 61bba849e..e56811561 100644
--- a/plugins/password/helpers/chpass-wrapper.py
+++ b/plugins/password/helpers/chpass-wrapper.py
@@ -11,12 +11,12 @@ BLACKLIST = (
 
 try:
     username, password = sys.stdin.readline().split(':', 1)
-except ValueError, e:
+except ValueError:
     sys.exit('Malformed input')
 
 try:
     user = pwd.getpwnam(username)
-except KeyError, e:
+except KeyError:
     sys.exit('No such user: %s' % username)
 
 if user.pw_uid < 1000:
-- 
2.24.1

